operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -95,11 +98,16 @@ protected Status createStatus() {
                 .onComplete(context.succeeding(v -> context.verify(() -> {
                     MeterRegistry registry = metrics.meterRegistry();
 
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations").meter().getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.successful").meter().getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.successful").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.failed").meter().getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.failed").tag("kind", "TestResource").counter().count(), is(0.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.locked").meter().getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.locked").tag("kind", "TestResource").counter().count(), is(0.0));
 
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.duration").meter().getId().getTags().get(1), is(Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")));
```

Comment:
You seem to repeat `Tag.of("selector", selectorLabels != null ? selectorLabels.toSelectorString() : "")` again and again. Maybe it is worth creating it once before all the asserts and just reuse it? That way, the asserts would be much more readable.

Reply:
@scholzj I have applied these suggestions